lib/model/time.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:sayilar/extensions/cases.dart';
+import 'package:sayilar/extensions/written_out.dart';
+
+/// A simple class for holding a time akin to [TimeOfDay].
+// TODO: It might make more sense to simply add a writtenOut extension to
+// TimeOfDay instead of creating a whole new class...
+class Time {
```

Comment:
It seems, that this can be replaced by the standard `TimeOfDay`, if I add a `writtenOut` and `toString` `extension` for it.